client/src/test/__mocks__/fileMock.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
+
+export class GenericComponent extends Component {
+  render() {
+    return (
+      <div data-test-id="generic">
+        <h1>Hello, World!</h1>
+      </div>
+    );
+  }
+}
```

Comment:
I wouldn't expect a `fileMock` to return a React component. It would probably be easier for this file to return the name of the file that it's mocking, so you can write test that expect the filename to appear. Here's how I've done it:

```js
const path = require('path');

module.exports = { process: (_, filename) => `module.exports = '${JSON.stringify(path.basename(filename))}';` };
```

Reply:
i think the naming convention here was incorrect. i was actually trying to create a generic component for the private route testing. i've renamed the file to reflect the intent of the code.

Reply:
OK, but you'll still need `fileMock` if any of your components import file extensions listed out in your jest config.

Reply:
Ok. I'll take care of that when I start working on the rest of the component testing.